admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.js

```diff
@@ -389,12 +389,15 @@ class SpecificPriceFormHandler {
    */
   enableSpecificPriceTaxFieldIfEligible(usePrefixForCreate) {
     const selectorPrefix = this.getPrefixSelector(usePrefixForCreate);
+    const spReductionSuffix = $(`${selectorPrefix}sp_reduction_type option:selected`).text();
 
     if ($(`${selectorPrefix}sp_reduction_type`).val() === 'percentage') {
       $(`${selectorPrefix}sp_reduction_tax`).hide();
     } else {
       $(`${selectorPrefix}sp_reduction_tax`).show();
     }
```

Comment:
```suggestion
    $(`${selectorPrefix}sp_reduction_tax`).toggle($(`${selectorPrefix}sp_reduction_type`).val() !== 'percentage');
```

Reply:
Hi @Progi1984 
Thanks man !
Could you explain please ?

Reply:
https://api.jquery.com/toggle/#toggle-display

Reply:
LGTM! But it's the old code ... As @matks  would say : _"I'm afraid of modifying the code of..."_ 😀
Thank you all the same !